backup_and_restore/application_backup_and_restore/features-and-plugins/oadp-configuring-velero-plugins.adoc

```diff
@@ -0,0 +1,73 @@
+:_content-type: ASSEMBLY
+[id="oadp-velero-plugins"]
+= OADP Velero plugins
+include::_attributes/common-attributes.adoc[]
+:context: oadp-features-plugins
+
+toc::[]
+
+You can configure two types of plugins when you install Velero:
+
+* Default cloud provider plugins
+* Custom plugins
+
+Both types of plugin are optional, but most users configure at least one cloud provider plugin.
+
+== Default Velero cloud provider plugins
```

Comment:
Anchor/ID string needed for all headings [per the docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings).

Reply:
Thanks done